DEVELOPMENT.md

```diff
@@ -0,0 +1,70 @@
+```
+psql plume
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO plume;
+GRANT USAGE, SELECT ON ALL SEQUENCES IN SCHEMA public TO plume;
+\q
+```
+#### Database Migration
+Now that the Postgres database is set up and the `plume` user has the privileges it  needs, we can set up the database using the diesel CLI. If this was your time installing Rust, you will probably need to run that using `cargo`. `cargo` is installed with `rustc` so if you followed the earlier instructions it will already be available.
+```
+cargo install diesel_cli
+```
+The first time you run this, you can run setup. After that, every time you pull the repository you will want to run the migration command in case there were any migrations. Those commands are
+```
+diesel setup --database-url='postgres://localhost/plume'
+diesel migration run --database-url='postgres://localhost/plume'
+```
+#### Configuration
+Now Plume should be running on your machine at [http://localhost:8000](http://localhost:8000). The first time you run the application, you'll want to configure your blog name on the [http://localhost:8000/configuration](http://localhost:8000/configuration) page. You'll be able to change this name later.
+
+#### Running Plume
+To run Plume locally, make sure you are once again in the Plume directory, such as `~/dev/Plume`. Now you will be able to run the application using the command
+```
+cargo run
+```
+#### Making a Pull Request
+To create an upstream fork of the repository in GitHub, click "Fork" in the top right button on the main page of the [Plume repository](https://github.com/Plume-org/Plume). Now, in the command line, set another remote for the repository by running the following command, replacing `myname` with the name under which you forked the repo. You can use another name besides `upstream` if you prefer
+```
+git remote add upstream https://github.com/myname/Plume.git
+```
+Now, make any changes to the code you want. After committing your changes, push to the upstream fork. Once your changes are made, visit the GitHub page for your fork and select "Make Pull Request". Add descriptive text, any issue numbers using hashtags to reference the issue number, screenshots of your changes if relevant, a description of how you tested your changes, and any other information that will help the project maintainers be able to quickly accept your pull requests.
+The project maintainers may suggestion further changes to improve the pull request even more. After implementing this locally, you can push to your upstream fork again and the changes will immediately show up in the pull request after pushing. Once all the suggested changes are made, the pull request may be accepted. Thanks for contributing.
```

Comment:
I don't know if you wanted to create a new paragraph here, but if you did, you should make sure to add a blank line between the two paragraphs.